zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: news_post
+title: "Ruby 1.9.3 的官方支持结束"
+author: "Olivier Lacan"
+translator: "Vincent Zhao"
+date: 2015-02-23 00:00:00 +0000
+lang: zh_cn
+---
+
+从今天开始，对 Ruby 1.9.3 的支持将全部结束。Ruby 新版的 Bug 和安全修复将不会再移植回 1.9.3。
+
+结束维护已经在[一年前预告](https://www.ruby-lang.org/zh_cn/news/2014/01/10/ruby-1-9-3-will-end-on-2015/)。
```

Comment:
@hlcfan you can leave comment here, see [here](https://help.github.com/articles/commenting-on-the-diff-of-a-pull-request/) for more information. In this case we already got a translation so no need to link back to `en` post. :smile: